### PR TITLE
Add OpenShift resources for 3.1.0

### PR DESCRIPTION
**Pull request by Chathuru**
Add OpenShift resources for 3.1.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
-# openshift-apim
```
Comment (ThilinaManamgoda) [inline README.md:1]: Why this line is removed?

File: openshift/am-pattern-2/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+	`oc create secret docker-registry wso2-image-pull-secret --docker-server=docker.wso2.com --docker-username=<SUBSCRIPTION_USERNAME> --docker-password=<SUBSCRIPTION_PASSWORD>`
+
+3. Deploy WSO2 APIM Server using the Template.yaml
+
+	`oc process -f Template.yaml -p NFS_SERVER_IP=<NFS_SERVER_IP> -p NFS_SHARE_DATABASE=<DATABASE_NFS_SHARE_PATH> -p NFS_SHARE_SYNAPSE=<SYNAPSE_NFS_SHARE_PATH> -p NFS_SHARE_EXECUTION_PLANS=<EXECUTION_PLANS_SHARE_PATH> -p NFS_SHARE_CARBON_DB_1=<CARBON_DB_1_NFS_SHARE_PATH> -p NFS_SHARE_SOLR_1=<SOLR_1_NFS_SHARE_PATH> -p NFS_SHARE_CARBON_DB_2=<CARBON_DB_2_NFS_SHARE_PATH> -p NFS_SHARE_SOLR_2=<SOLR_2_NFS_SHARE_PATH> -p OPENSHIFT_BASE_DOMAIN=<OPENSHIFT_URL> | oc create -f -`
+
+4. Access product management consoles. Obtain the (HOST/PORT) of the route resource
+
+	`oc get route -n <NAMESPACE>`
```
Comment (ThilinaManamgoda) [inline openshift/am-pattern-2/README.md:32]: Is it a namespace or project?

**Review by ThilinaManamgoda — CHANGES_REQUESTED** [review]

File: openshift/am-pattern-2/README.md (hunk 2 above)
Comment (Chathuru) [reply openshift/am-pattern-2/README.md]: Fix in https://github.com/wso2/openshift-apim/pull/1/commits/37ccb91b0e914a3c7390a4a610df08202b3226f9 commit. Change namespace to project as its the terminology used in openshift

File: README.md (hunk 1 above)
Comment (Chathuru) [reply README.md:1]: Rewrite the README file to tally with helm readme file

**Pull request merged**